config/sync/facets.facet.type_of_food.yml

```diff
@@ -0,0 +1,75 @@
+uuid: 8bbabce3-d9eb-48f3-84e1-e933989a3c19
+langcode: en
+status: true
+dependencies:
+  config:
+    - search_api.index.default_index
+    - views.view.catalog
+  module:
+    - search_api
+id: type_of_food
+name: 'Type of food'
+url_alias: type_of_food
+weight: 0
+min_count: 1
+show_only_one_result: false
+field_identifier: field_type_of_food
+facet_source_id: 'search_api:views_page__catalog__page_1'
+widget:
+  type: checkbox
+  config:
+    show_numbers: true
+    soft_limit: 0
+    soft_limit_settings:
+      show_less_label: 'Show less'
+      show_more_label: 'Show more'
+    show_reset_link: false
+    reset_text: 'Show all'
+    hide_reset_when_no_selection: false
+query_operator: or
+use_hierarchy: false
+expand_hierarchy: false
+enable_parent_when_child_gets_disabled: true
+hard_limit: 0
+exclude: false
+only_visible_when_facet_source_is_visible: true
+processor_configs:
+  active_widget_order:
+    processor_id: active_widget_order
+    weights:
+      sort: 20
+    settings:
+      sort: DESC
+  count_widget_order:
+    processor_id: count_widget_order
+    weights:
+      sort: 30
+    settings:
+      sort: DESC
+  display_value_widget_order:
+    processor_id: display_value_widget_order
+    weights:
+      sort: 40
+    settings:
+      sort: ASC
+  list_item:
+    processor_id: list_item
+    weights:
+      build: 5
+    settings: {  }
+  translate_entity:
+    processor_id: translate_entity
+    weights:
+      build: 5
+    settings: {  }
+  url_processor_handler:
+    processor_id: url_processor_handler
+    weights:
+      pre_query: 50
+      build: 15
+    settings: {  }
+empty_behavior:
+  behavior: text
+  text_format: plain_text
+  text: 'Facet should be here.'
```

Comment:
From a UX perspective this message is not too helpful. I think I'd rather hide the facet if there are no results to filter.

![Screenshot_20200416_123531](https://user-images.githubusercontent.com/13573052/79446919-56dd4e00-7fdf-11ea-81fe-13de395f70c8.png)

Reply:
I used this as sort of a debugging tool: to see that the facet is in the right place even if it doesn't work yet, or it's empty. Of course I can hide it if necessary.

Reply:
Fixed!